### ISSUE TITLE
Bug: MFTF does not close connection to Selenium hub

### ISSUE THREAD

**Issue by lbajsarowicz** [magento/magento2-functional-testing-framework#660]
While everything works as expected when running standalone instance of Selenium, when running Selenium hub repeatedly - the browser instances are not being freed up.

### Preconditions
1. Start selenium hub and 3 instances of selenium chrome assigned to the hub

### Steps to reproduce
1. Run any suite and wait till it's done
2. Try to run the suite again

### Expected result
1. Suite is being performed

### Actual result
1. MFTF reports that it cannot connect to selenium hub
![image](https://user-images.githubusercontent.com/1639941/77830946-6f6aee80-712c-11ea-8c55-1c2c782079c0.png)

**Comment by lbajsarowicz** [comment]
Having 3 instances of Selenium Chrome and running single test, I end up with `$ curl http://selenium-hub:4444/wd/hub/status` returning
```json
{
  "status": 0,
  "value": {
    "ready": false,
    "message": "No spare hub capacity",
    "build": {
      "revision": "e82be7d358",
      "time": "2018-11-14T08:25:53",
      "version": "3.141.59"
    },
    "os": {
      "arch": "amd64",
      "name": "Linux",
      "version": "5.3.0-40-generic"
    },
    "java": {
      "version": "1.8.0_242"
    }
  }

```

**Comment by lbajsarowicz** [comment]
That might be a hint: https://github.com/elgalu/docker-selenium/issues/296

**Comment by jilu1** [comment]
@lbajsarowicz This issue might have been fixed. Can you please try this with MFTF 3.0.0?

**Comment by lbajsarowicz** [comment]
@jilu1 Verifying that now.

**Comment by lbajsarowicz** [comment]
@jilu1 Looks like it was fixed. Could you tell me which PR fixed that?

**Comment by jilu1** [comment]
> @jilu1 Looks like it was fixed. Could you tell me which PR fixed that?

It's fixed as part of 3.0.0.
